### ISSUE TITLE
have a "if you use this service in research, plz cite us" note in BridgeDb OpenAPI UI

### ISSUE THREAD

**Issue by egonw** [bridgedb/BridgeDb#82]

**Comment by egonw** [comment]
@nunogit, is it feasible to make "branding" changes to the OpenAPI UI?

**Comment by nunogit** [comment]
@egonw I am not sure I understand what you mean.
It's feasible to do changes to the UI. Colors, logos, etc.
Probably, also, make it open API, as we are still using swagger. 
Do I answer your question?

**Comment by egonw** [comment]
Yes, sort of. The follow question is, of course, how do we practically go about to do this? What file needs to be changed? Which git repo holds that file?

**Comment by nunogit** [comment]
Most likely the javascript on index.html. But we might be able to change the index.html only, depending on what you want to do.

https://github.com/bridgedb/swagger-webservice/blob/master/index.html

Where do you want to put this? I guess it should have some highlight.
